src/js/yeldaWebchatInjector.js

```diff
@@ -761,29 +767,38 @@ class YeldaChat {
   unLoadChat() {
     this.toggleFrameListener(true)
 
-    /**
-     * If init or setupChat has been called multiple times we might end up with multiple yelda_iframe_container and yelda_container
-     * So to be sure that the destroy the webchat window completely, let's find all the matching elements and remove them all
-     */
-    if (this.iframeContainer) {
-      for (const element of document.querySelectorAll("[id='yelda_iframe_container']")) {
-        element.remove()
+    if (this.assistantImage) {
+      // If bubbleContainer exists instead of removing the assistantImage dom, replace with the original dom of bubbleContainer
+      if (this.bubbleContainer) {
+        this.assistantImage.removeAttribute('id', 'yelda_assistant_img')
+        this.assistantImage.removeAttribute('class', 'yelda_assistant_img default')
+        this.assistantImage.removeEventListener('click', this.openChat)
+        this.assistantImage.replaceWith(this.bubbleContainerClone)
```

Comment:
once this.assistantImage has been replaced by bubbleContainerClone, maybe we could reset the bubbleContainerClone ?

Reply:
bubbleContainerClone is reset to null later in the same function